problems/removeEvenStrings.js

```diff
@@ -6,8 +6,10 @@
 * @returns {string[]} - Returns the strings in arr that have an odd number of characters
 */
 
-function removeEvenStrings() {
-
-}
+function removeEvenStrings (arr) {
+    return arr.filter((arr)=>{
```

Comment:
This is really close. Should be using str instead of arr inside of callback, and checking arr.length % 2 === 0